gr-zeromq/lib/base_impl.cc

```diff
@@ -192,6 +210,23 @@ bool base_source_impl::load_message(bool wait)
     d_socket->recv(&d_msg);
 #endif
 
+    /* Throw away key and get the first message. Avoid blocking if a multi-part
+     * message is not sent */
+    if (d_key.size() > 0 && !more) {
+        int64_t is_multipart;
+        d_socket->getsockopt(ZMQ_RCVMORE, &is_multipart, &more_len);
+
```

Comment:
guessing there is an extra whitespace here

Reply:
thanks, I'll remove the extra line